libtcmu.c

```diff
@@ -177,8 +177,8 @@ static int reconfig_device(struct tcmulib_context *ctx, char *dev_name,
 
 	ret = dev->handler->reconfig(dev, &cfg);
 	if (ret < 0) {
-		tcmu_dev_err(dev, "Handler reconfig failed with error %d.\n",
-			     ret);
+		tcmu_dev_dbg(dev, "Handler reconfig failed with error %s.\n",
+		             strerror(-ret));
```

Comment:
@pkalever @mikechristie 

Will it make sense to print the cfg.type here like:

```
+               tcmu_dev_dbg(dev, "Handler reconfig for %u failed with error %d.\n",
+                            (unsigned int)cfg.type, ret);
```

Or we must guess what the unsupport type is every time.

Thanks.

Reply:
With the latest kernel which has support the write_cache and dev_config attrs, when restarting the tcmu-runner it will be:

2019-06-22 12:08:14.534 9999 [INFO] tcmu_glfs_reconfig:689: TCMULIB_CFG_DEV_CFGSTR not sup!
2019-06-22 12:08:14.534 9999 [ERROR] reconfig_device:196 glfs/block3: Handler reconfig failed with error -95.
2019-06-22 12:08:14.534 9999 [INFO] tcmu_glfs_reconfig:692: TCMULIB_CFG_WRITE_CACHE not sup!
2019-06-22 12:08:14.534 9999 [ERROR] reconfig_device:196 glfs/block3: Handler reconfig failed with error -95.

Reply:
Yeah.

It is ok if pkalever wants to sneak it in here or we can do it in another patch/PR since pkalever's original goal of this PR/patch was making the logs nicer from a flooding standpoint.

Reply:
I am okay with this.
Thanks.